backend/src/settings.py

```diff
@@ -118,3 +93,15 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
+
+
+import dynaconf  # noqa
```

Comment:
Let's use all import in the beginning of the file with the following order:
 - Python packages
 - 3rd party packages
 - Our modules